content/docs/command-reference/fetch.md

```diff
@@ -17,22 +17,12 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+Downloads tracked data from remote storage into the <abbr>cache</abbr> (without
+placing it in the <abbr>workspace</abbr> like `dvc pull`). This makes them
```

Comment:
placing -> changing the workspace?

Reply:
Not sure. This parenthesis is just in case the user didn't catch the "into the cache" part so if they didn't catch that, "without changing the workspace" will be confusing. Safer to be explicit?